docs/software/communication/openmpi.md

```diff
@@ -32,27 +22,55 @@ Additionally, the following environment variables should be set:
 ```bash
 export PMIX_MCA_psec="native" # (1)!
 export FI_PROVIDER="cxi" # (2)!
-export OMPI_MCA_pml="^ucx" # (3)!
+export OMPI_MCA_pml="cm" # (3)!
 export OMPI_MCA_mtl="ofi" # (4)!
 ```
 
 1. Ensures PMIx uses the same security domain as Slurm. Otherwise PMIx will print warnings at startup.
 2. Use the CXI (Slingshot) provider.
-3. Use anything except [UCX](https://openucx.org/documentation/) for [point-to-point communication](https://docs.open-mpi.org/en/v5.0.x/mca.html#selecting-which-open-mpi-components-are-used-at-run-time). The `^` signals that OpenMPI should exclude all listed components.
+3. Use CM for [point-to-point communication](https://docs.open-mpi.org/en/v5.0.x/mca.html#selecting-which-open-mpi-components-are-used-at-run-time).
 4. Use libfabric for the [Matching Transport Layer](https://docs.open-mpi.org/en/v5.0.x/mca.html#frameworks).
 
 !!! info "CXI provider does all communication through the network interface cards (NICs)"
     When using the libfabric CXI provider, all communication goes through NICs, including intra-node communication.
-    This means that intra-node communication can not make use of shared memory optimizations and the maximum bandwidth will not be severely limited.
+    This means that intra-node communication can not make use of shared memory optimizations and the maximum bandwidth will be severely limited.
+
+### Using the experimental LINKx provider
+
+The default configuration routes all communication through the NICs.
+While performance may sometimes be acceptable, this mode does not make full use of the much higher intra-node bandwidth available on Grace-Hopper nodes.
+In particular, GPU-GPU communication is significantly faster when using the appropriate intra-node links.
+
+The experimental [LINKx](https://ofiwg.github.io/libfabric/v2.3.1/man/fi_lnx.7.html) libfabric provider allows composing multiple libfabric providers for inter- and intra-node communication.
+The CXI provider can be used for inter-node communication while the shared memory (`shm`) provider can be used to take advantage of xpmem for CPU-CPU communication and GDRCopy for GPU-GPU communication.
+
+!!! warning "The LINKx provider is experimental and may contain bugs, in particular for intra-node communication"
```

Comment:
```suggestion
!!! danger "The LINKx provider is experimental and may contain bugs, in particular for intra-node communication"
```

Reply:
Given e.g. the problems @jpcoles-cscs had in his tests with lnx, I'll keep it as the potentially-better-but-secondary option and make it a `danger` as you proposed. I'll slightly restructure the intro though to present the two options a bit more eqally.